### ISSUE TITLE
gem uninstall won't work on windows 10

### ISSUE THREAD

**Issue by rellampec** [rubygems/rubygems#2866]
I'm having a problem.

My current problem is:

- when I try to uninstall a gem version or a full gem it does not give me even an error message, and the gem is still in the list of installed gems

I have tried the following commands, but none give me the successfully uninstalled message (just prompts to the next input line):

```
gem uninstall gem-name
gem uninstall -a gem-name
gem uninstall gem-name -v 0.5.X
gem uninstall gem-name --install-dir C:\Ruby\lib\ruby\gems\2.4.0
# the above as per result on 'gem list -d gem-name
gem uninstall gem-name --install-dir C:/Ruby/lib/ruby/gems/2.4.0
bundle exec gem uninstall -a gem-name
```

The bash used doesn't really matter (tried with git bash, windows `cmd`, PowerShell).

This issue is related to:
  - [x] The command line `gem`
 

Here are my current environment details:

```
$ gem env version
3.0.4
```

I will abide by the [code of conduct](https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md).

**Comment by MSP-Greg** [comment]
I updated my Windows Ruby 2.4.6 install to RG 3.0.4, and I could uninstall a gem, but trying the same on a default gem exhibited the behavior you're mentioning (no output, immediate return to command line).  Hence, I couldn't repo.

Not sure what your issue is, but a message like 'can't uninstall default gem' might be helpful...

**Comment by deivid-rodriguez** [comment]
This was fixed in https://github.com/rubygems/rubygems/pull/2707, and it's set to be released in 3.0.5 according to the milestone of that PR.

Closing!
